packages/infra/lib/api-stack/api-service.ts

```diff
@@ -182,6 +183,8 @@ export function createAPIService({
           CXS_WITH_CQ_DIRECT_FEATURE_FLAG: appConfigEnvVars.cxsWithCQDirectFeatureFlag,
           CXS_WITH_ENHANCED_COVERAGE_FEATURE_FLAG:
             appConfigEnvVars.cxsWithEnhancedCoverageFeatureFlag,
+          CXS_WITH_INCREASED_PATIENT_LIMIT_FEATURE_FLAG:
+            appConfigEnvVars.cxsWithIncreasedSandboxLimitFeatureFlag,
```

Comment:
For consistency i would name the variables either `CXS_WITH_INCREASED_PATIENT_LIMIT_FEATURE_FLAG` & `cxsWithIncreasedPatientLimitFeatureFlag` or `CXS_WITH_INCREASED_SANDBOX_LIMIT_FEATURE_FLAG` & `cxsWithIncreasedSandboxLimitFeatureFlag`